api/src/main/java/org/openmrs/Order.java

```diff
@@ -403,7 +428,7 @@ public boolean isActive() {
 	/**
 	 * Convenience method to determine if the order is active as of the specified date
 	 * 
-	 * @param checkDate - the date on which to check order. if null, will use current date
+	 * @param aCheckDate - the date on which to check order. if null, will use current date
```

Comment:
Are these some stray changes?

Reply:
Agree with @wluyima , this looks like a typo (and repeated below)